React/tic-tac-toe/src/utils/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = sessionStorage.getItem('state');
```

Comment:
lets use a more descriptive word for what you are saving in the storage.. `state` is very general